examples/opensora_hpcai/opensora/models/layers/blocks.py

```diff
@@ -287,7 +286,7 @@ def construct(self, x, mask=None):
         # reshape FA output to original attn input format (b n h*d)
         out = out.view(B, N, -1)
 
-        return self.proj_drop(self.proj(out)).to(x_dtype)
+        return self.proj_drop(self.proj(out))
```

Comment:
This will cause a problem in FP32 training as FA converts attention's type to BF16.

Reply:
fixed